publication-commons/src/test/java/no/unit/nva/publication/model/business/ResourceTest.java

```diff
@@ -48,7 +49,19 @@ void copyContainsAllFields(Class<?> publicationInstanceType) {
         assertThat(copy, is(equalTo(resource)));
         assertThat(resourceJson, is(equalTo(copyJson)));
     }
-    
+
+    @ParameterizedTest
+    @MethodSource("publicationInstanceProvider")
+    void copyShouldCreateDeepCopyWithoutOverridingValuesOfOriginalResourceWhenModifyingCopy(
+        Class<?> publicationInstanceType) {
+        Resource resource = sampleResource(publicationInstanceType);
+        Resource copy = resource.copy().build();
+        copy.setLink(randomUri());
+        assertThat(resource, not(is(equalTo(copy))));
+        Diff diff = javers.compare(resource, copy);
+        assertThat(diff.prettyPrint(), diff.getChanges().size(), is(1));
```

Comment:
Her ville jeg vært mer verbos.
Assert at de to publikasjonene er like før du endrer på link.
Og ta med flere felt etterpå, hvertfall noe felt som er ett par nivåer nedi entitydescription.